vsc-extension-quickstart.md

```diff
@@ -1,41 +0,0 @@
-# Welcome to your VS Code Extension
```

Comment:
can't believe this file is still here 😂

Reply:
Heh really? Can't approve my point that these project creating tools add a lot of (unnecessary) files.